### PR TITLE
update consensus and finality page

### PR DESCRIPTION
**Pull request by eshaben**
### Description

This PR updates the consensus and finality page to include snippets on using `moon_isTxFinalized` and it rearranges the content a bit so it flows a little better.

### Checklist

- [x] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [x] If pages have been moved around, I have run the `move-pages.py` script to move the pages and update the image paths on the chinese repo
    - [x] After the script has been run, I have created an additional PR in `moonbeam-docs-cn`

### Corresponding PRs

https://github.com/moonbeam-foundation/moonbeam-docs-cn/pull/357

### After Translation Requirements

- [x] No additional PRs are required after the translations are done

### CODE REVIEW

File: builders/get-started/eth-compare/consensus-finality.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ In terms of finality, blocks on Ethereum can take quite a bit longer to finalize
 
 Moonbeam does not use checkpoint blocks and instead relies on Polkadot's GRANDPA finality gadget, where the finality process is completed in parallel to block production. In addition, the finality process incorporates the blockchain's structure, which allows the relay chain validators to vote on the highest block that they think is valid. In this scenario, the vote would apply to all of the blocks leading up to the one that is finalized, which speeds up the finalization process. After a block has been included in the relay chain, a block can be finalized within one block on Moonbeam.
 
-## Strategy to Check Transaction Finality {: #strategy-to-check-tx-finality }
+## Check Transaction Finality with Ethereum RPC Endpoints {: #strategy-to-check-tx-finality }
```
Comment (themacexpert) [inline builders/get-started/eth-compare/consensus-finality.md:42]: Shouldn't we have the tags match exactly or pretty close to the title? e.g. #check-transaction-finality

File: builders/get-started/eth-compare/consensus-finality.md [hunk 2]
```diff
@@ -83,36 +73,64 @@ The snippets below follow the strategy outlined in the [previous section](#strat
     --8<-- 'code/builders/get-started/eth-compare/consensus-finality/web3.py'
     ```
 
-You can modify these scripts to use `moon_isBlockFinalized` and `moon_isTxFinalized`. To do this, you can make custom calls to the Substrate JSON-RPC using the `send` method of both [Web3.js](https://web3js.readthedocs.io/){target=_blank} and [Ethers.js](https://docs.ethers.org/){target=_blank}. Custom RPC requests are also possible using [Web3.py](https://web3py.readthedocs.io/){target=_blank} with the `make_request` method. You'll need to pass in the method name and the parameters to the custom request, which you can find on the [Moonbeam Custom API](/builders/build/moonbeam-custom-api/){target=_blank} page.
+## Check Transaction Finality with Moonbeam RPC Endpoints {: #checking-tx-finality-with-moonbeam-rpc-endpoints }
```
Comment (themacexpert) [inline builders/get-started/eth-compare/consensus-finality.md:76]: Check / checking mismatch between title and tag could be confusing

File: builders/get-started/eth-compare/consensus-finality.md [hunk 3]
```diff
@@ -83,36 +73,64 @@ The snippets below follow the strategy outlined in the [previous section](#strat
...
 
-=== "Ethers.js"
+Moonbeam has added support for two custom RPC endpoints, `moon_isBlockFinalized` and `moon_isTxFinalized`, that can be used to check whether an on-chain event is finalized. These methods are a bit more straight forward, as you don't need to compare block numbers to ensure your transaction was finalized.
```
Comment (themacexpert) [inline builders/get-started/eth-compare/consensus-finality.md:78]:
```suggestion
Moonbeam has added support for two custom RPC endpoints, `moon_isBlockFinalized` and `moon_isTxFinalized`, that can be used to check whether an on-chain event is finalized. These methods are a bit more straightforward, as you don't need to compare block numbers to ensure your transaction was finalized.
```

File: builders/get-started/eth-compare/consensus-finality.md [hunk 4]
```diff
@@ -83,36 +73,64 @@ The snippets below follow the strategy outlined in the [previous section](#strat
...
+
+    === "Web3.py"
+
+        ```py
+        --8<-- 'code/builders/get-started/eth-compare/consensus-finality/custom-rpc/tx/web3.py'
+        ```
+
+## Check Transaction Finality with Substrate RPC Endpoints {: #checking-tx-finality-with-substrate-rpc }
+
+Using the following three RPC requests from the Substrate JSON-RPC, you can fetch the current finalized block and compare it with the block nunmber of the transaction you want to check finality for:
```
Comment (themacexpert) [inline builders/get-started/eth-compare/consensus-finality.md:126]:
```suggestion
Using the following three RPC requests from the Substrate JSON-RPC, you can fetch the current finalized block and compare it with the block number of the transaction you want to check finality for:
```

**Review by themacexpert — APPROVED** [review]
👍

**Comment by eshaben** [comment]
Updated header attributes here and also updated (where it makes sense) in the chinese PR. Thanks for the review!

**Pull request merged**